Client/Frontend/Browser/PageZoom/PageZoomView.swift

```diff
@@ -0,0 +1,182 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import WebKit
+import Data
+import Shared
+import BraveShared
+
+private struct ZoomView: View {
+  @ScaledMetric private var buttonWidth = 44.0
+  
+  var minValue: Double
+  var maxValue: Double
+  @Binding var value: Double
+  
+  var onDecrement: () -> Void
+  var onReset: () -> Void
+  var onIncrement: () -> Void
+  
+  var body: some View {
+    HStack(spacing: 0.0) {
+      Button(action: onDecrement) {
+        Image(systemName: "minus")
+          .font(.system(.footnote).weight(.medium))
+          .foregroundColor(Color(UIColor.braveLabel))
+          .frame(width: buttonWidth)
+      }
+      .disabled(value == minValue)
+      
+      Divider()
+      
+      Text(NSNumber(value: 0.999), formatter: PageZoomView.percentFormatter)
```

Comment:
Added a comment. But `999` is the largest 3 digit number in terms of character widths. So if the percentage goes from 50 (2 characters) to 100 (3 characters) then to 300 (3 characters), it means 50 is smaller than 100 width-wise, and 100 is smaller than 300 width-wise because 3 takes up more space than 1. But if I put `999` a fixed size, no matter what the number is, the size of the text there doesn't shift and glitch.